tjmonopix2/system/scan_base.py

```diff
@@ -147,10 +147,14 @@ def __repr__(self):
 
 
 class ScanBase(object):
-    '''
-        Basic run meta class.
+    """ Basic run meta class.
         Base class for scan- / tune- / analyze-class.
-    '''
+
+    Parameters
+    ----------
+    object : object
+        Used for scan configurations
+    """
```

Comment:
Same as for the `FIFO` class. `object` can be removed and it also no parameter anymore. Everything in python is an object anyway ;)